lib/foreman_remote_execution_core/version.rb

```diff
@@ -0,0 +1,3 @@
+module ForemanRemoteExecutionCore
+  VERSION = '0.0.1'
```

Comment:
I would make it 1.0.0, so that the first number matches the rex version, but I would probably not try to keep the version of rex and rex-core to be in sync.